packages/server/graphql/mutations/helpers/handleCompletedStage.ts

```diff
@@ -60,6 +62,17 @@ const handleCompletedRetrospectiveStage = async (
       )
 
       data.reflectionGroups = sortedReflectionGroups
+    } else if (stage.phaseType === GROUP) {
+      const {phases} = meeting
+      unlockAllStagesForPhase(phases, 'discuss', true)
+      await r
+        .table('NewMeeting')
+        .get(meeting.id)
+        .update({
+          phases
+        })
+        .run()
+      data.meeting = meeting
```

Comment:
+1 is this necessary? looks like unlockAllStagesForPhase _should_ mutate the phases object directly `data.meeting === meeting`, but I could be missing something

Reply:
`unlockAllStagesForPhase` mutates the phase object, but we need to add the updated meeting to the payload, otherwise we get: `Cannot return null for non-nullable field GroupPhaseCompletePayload.meeting`

Reply:
`unlockAllStagesForPhase` mutates the phase object, but we need to add the updated meeting to the payload, otherwise we get: `Cannot return null for non-nullable field GroupPhaseCompletePayload.meeting`

Reply:
`unlockAllStagesForPhase` mutates the phase object, but we need to add the updated meeting to the payload, otherwise we get: `Cannot return null for non-nullable field GroupPhaseCompletePayload.meeting`